syncano_cli/sync/project.py

```diff
@@ -80,6 +81,7 @@ def push_to_instance(self, instance, all=False, classes=None,
             push_classes(instance, sync_classes)
         elif not scripts:
             LOG.info('Nothing to sync.')
+            click.echo('Nothing to sync.')
```

Comment:
Not, the second is for test assert - will try to figure it out, how to leave only one.